docs/docs/cmd/spo/tenant/tenant-settings-set.md

```diff
@@ -258,6 +258,12 @@ m365 spo tenant settings set [options]
 `--DisableCustomAppAuthentication [DisableCustomAppAuthentication]`
 : Configure if ACS-based app-only auth should be disabled or not. Allowed values `true,false`
 
+`--EnableAzureADB2BIntegration [EnableAzureADB2BIntegration]`
+: Enables the preview for OneDrive and SharePoint integration with Azure AD B2B. Allowed values `true,false`. Azure AD one-time passcode needs to be enabled for this integration to work. For more information see [http://aka.ms/spo-b2b-integration](http://aka.ms/spo-b2b-integration).
+
+`--SyncAadB2BManagementPolicy [SyncAadB2BManagementPolicy]`
+: Syncs Azure B2B Management Policies. Allowed values `true,false`. For more information, see [SharePoint and OneDrive integration with Azure AD B2B](https://docs.microsoft.com/en-us/sharepoint/sharepoint-azureb2b-integration-preview).
```

Comment:
Both links point to the same page. Let's just one of them and use it consistently